pages/join/03-company.tsx

```diff
@@ -161,156 +160,146 @@ export default function JoinStep3({ industries, pageTitle }) {
-                  gridColumn: `span ${columnCount}`,
-                }}
-              >
-                <Selectable
-                  headline={"N/A, or Prefer Not to Answer"}
                   fullWidth
-                  disabled={isMaxSelected && !deferIndustry}
-                  selected={deferIndustry === "true"}
-                  onClick={() =>
-                    setDeferIndustry(
-                      deferIndustry === "true" ? undefined : "true"
-                    )
+                  centered
+                  variant={SelectableVariant.Blank}
+                  onClear={
+                    industrySuggested && !deferIndustry
+                      ? () =>
+                          window.confirm(
+                            "Are you sure you want to clear this field?"
+                          ) && setIndustrySuggested("")
+                      : undefined
                   }
                 />
-                {showSuggestButton ? (
-                  <Selectable
-                    headline={
-                      industrySuggested
-                        ? `${industrySuggested}`
-                        : "+ Add industry"
-                    }
-                    onClick={() => setShowSuggestButton(false)}
-                    selected={!!industrySuggested && !deferIndustry}
-                    disabled={
-                      (isMaxSelected && !!!industrySuggested) ||
-                      deferIndustry === "true"
-                    }
-                    fullWidth
-                    centered
-                    variant={SelectableVariant.Alt}
-                    onClear={
-                      industrySuggested && !deferIndustry
-                        ? () =>
-                            window.confirm(
-                              "Are you sure you want to clear this field?"
-                            ) && setIndustrySuggested("")
-                        : undefined
-                    }
-                  />
-                ) : (
-                  <InputBox
-                    fullWidth
-                    border
-                    focusedOnInit
-                    onChange={(e) => setIndustrySuggested(e.target.value)}
-                    onBlur={() => setShowSuggestButton(true)}
-                    onEnter={() => setShowSuggestButton(true)}
-                    value={industrySuggested}
-                    disabled={isMaxSelected && !!!industrySuggested}
-                  />
-                )}
-              </div>
-            </SelectableGrid>
-          </div>
-          {!showSuggestButton || industrySuggested
-            ? WorkExperienceWarning()
-            : null}
+              ) : (
+                <Input
+                  autoFocus
+                  name="add-field"
+                  centered
+                  fullHeight
+                  onChange={(e) => {
+                    setIndustrySuggested(e.target.value + "fda");
```

Comment:
errrr is this `fda` thing supposed to be here? it's adding this unexpected text when I'm typing "test"

![Jun-30-2023 18-02-50](https://github.com/hawaiians/hawaiiansintech/assets/26368729/f1a0c47c-d6ed-440e-b39e-81c401b16343)